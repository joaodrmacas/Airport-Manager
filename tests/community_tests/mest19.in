a LIS Portugal		Lisbon Lisboa
a OPO Portugal		Porto
a FAO Portugal		Faro
v TP12 LIS OPO 23-03-2022 10:00 00:56 100
v TP222 LIS OPO 24-03-2022 10:00 00:50 50
v TP12 LIS OPO 24-03-2022 11:00 00:40 75
v TP333 FAO OPO 25-03-2022 22:00 00:55 110
v TP444 LIS OPO 25-03-2022 23:00 00:50 140
v
e TP12
v
q
