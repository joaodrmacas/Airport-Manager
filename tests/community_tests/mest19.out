airport LIS
airport OPO
airport FAO
TP12 LIS OPO 23-03-2022 10:00
TP222 LIS OPO 24-03-2022 10:00
TP12 LIS OPO 24-03-2022 11:00
TP333 FAO OPO 25-03-2022 22:00
TP444 LIS OPO 25-03-2022 23:00
TP222 LIS OPO 24-03-2022 10:00
TP333 FAO OPO 25-03-2022 22:00
TP444 LIS OPO 25-03-2022 23:00
